components/metrics/lib.rs

```diff
@@ -96,7 +93,7 @@ pub fn set_metric<U: ProgressiveWebMetric>(
 }
 
 // https://github.com/GoogleChrome/lighthouse/issues/27
-#[derive(HeapSizeOf)]
+#[ignore_heap_size_of = "can't measure channels"]
```

Comment:
This doesn't work. We should leave the derive, but add `#[ignore_heap_size_of = "can't measure channels"]` on the `time_profiler_chan` member instead.